example_custom_unpacking_test.go

```diff
@@ -118,8 +118,8 @@ func Example_customUnpacking() {
 	fmt.Println("Tuples (tuples2):", tuples2)
 
 	// Call a function "func_name" returning a table of custom tuples.
-	var tuples3 [][]Tuple3
-	err = conn.Call17Typed("func_name", []interface{}{}, &tuples3)
+	var tuples3 []Tuple3
+	err = conn.Call16Typed("func_name", []interface{}{}, &tuples3)
```

Comment:
What I really don't like about CALL_16: it implicitly changes a shape of the result (and the rules are not very obvious). Personally I would avoid it as much as possible. Or at least call only those functions, which return exactly what we'll receive on the connector side: so it will not be wrapped into an array or unwrapped. Maybe skipping of some tests is not so evil as pollute them with CALL_16.

I don't insist, just share my feeling about this.